### PR TITLE
[PartDesign] Expose WizardShaft/ strings to translation

### PR DESCRIPTION
**Pull request by luzpaz**
Expose missing PD WizardShaft strings to crowdin

### CODE REVIEW

**Comment by luzpaz** [comment]
@chennes can you take a look at this when you get a chance?

CC @kaktusus

File: src/Mod/PartDesign/WizardShaft/WizardShaftTable.py [hunk 1]
```diff
@@ -40,15 +40,15 @@ class WizardShaftTable:
...
-              ]    
+               "QT_TRANSLATE_NOOP("WizardShaftTable", "Length [mm]"),
+               "QT_TRANSLATE_NOOP("WizardShaftTable", "Diameter [mm]"),
+               "QT_TRANSLATE_NOOP("WizardShaftTable", "Inner diameter [mm]"),
+               "QT_TRANSLATE_NOOP("WizardShaftTable", "Constraint type"),
+               "QT_TRANSLATE_NOOP("WizardShaftTable", "Start edge type"),
+               "QT_TRANSLATE_NOOP("WizardShaftTable", "Start edge size"),
+               "QT_TRANSLATE_NOOP("WizardShaftTable", "End edge type"),
+               "QT_TRANSLATE_NOOP("WizardShaftTable", "End edge size)"
+              ]
```
Comment (AjinkyaDahale) [inline src/Mod/PartDesign/WizardShaft/WizardShaftTable.py:51]: Likely quotation mark discrepancy

Comment (luzpaz) [reply src/Mod/PartDesign/WizardShaft/WizardShaftTable.py:51]: Missing closing double quotes after closing parenthesis, right?

Comment (AjinkyaDahale) [reply src/Mod/PartDesign/WizardShaft/WizardShaftTable.py:51]: I was thinking more on the lines of removing quotation before `QT_TRANSLATE_NOOP`.
Plus at line 50 there should be a quote end before the parenthesis.

Comment (luzpaz) [reply src/Mod/PartDesign/WizardShaft/WizardShaftTable.py]: Fixed in c114a7a  ?

Comment (AjinkyaDahale) [reply src/Mod/PartDesign/WizardShaft/WizardShaftTable.py]: Perhaps an `import`?

Comment (AjinkyaDahale) [reply src/Mod/PartDesign/WizardShaft/WizardShaftTable.py]: Fixed by using `QtCore.QT_...`

File: src/Mod/PartDesign/WizardShaft/WizardShaftTable.py [hunk 2]
```diff
@@ -40,15 +40,15 @@ class WizardShaftTable:
...
-                "Length [mm]",
-               "Diameter [mm]",
-               "Inner diameter [mm]",
-               "Constraint type",
-               "Start edge type",
-               "Start edge size",
-               "End edge type",
-               "End edge size"
-              ]    
+               QT_TRANSLATE_NOOP("WizardShaftTable", "Length [mm]"),
```
Comment (chennes) [inline src/Mod/PartDesign/WizardShaft/WizardShaftTable.py:43]: For this and all other lines:
```suggestion
               QtCore.QT_TRANSLATE_NOOP("WizardShaftTable", "Length [mm]"),
```

**Review by chennes — CHANGES_REQUESTED** [review]
There are some changes required before this code will run, and a couple of these lines shouldn't be translated -- the code flow depends on the exact word. In addition, there are a few other items that could be exposed to translation (window titles, column headers, etc.).

**Review by chennes — APPROVED** [review]
With the latest changes, this looks good to me.

**Comment by luzpaz** [comment]
Thank you so very much @AjinkyaDahale and @chennes !  
@kkremitzki this is ready for merge

**Pull request merged**